templates/master.cf.common.erb

```diff
@@ -15,7 +15,7 @@ amavis unix - - - - 2 smtp
         -o mynetworks=127.0.0.0/8
         -o strict_rfc821_envelopes=yes
         -o receive_override_options=no_unknown_recipient_checks,no_header_body_checks
-        -o smtpd_bind_address=127.0.0.1
```

Comment:
This PR does two things. Adds an amavis processes param and also fixes a bug in the config.
Would you like it separated into two PRs?